.github/workflows/schedule_candidate_release.yml

```diff
@@ -13,10 +13,17 @@ jobs:
     # Don't run this in everyone's forks.
     if: github.repository == 'google/iree'
     steps:
+      - name: Get the last green commit
+        id: last_green_commit
+        uses: talentpair/last-green-commit-action@8b4b3bcd4ab5d9ab16875ce20ae012c4ce47ae68
+        with:
+          github-token: ${{ secrets.WRITE_ACCESS_TOKEN }}
```

Comment:
I don't think you should need write access to do this read operation :-) Does GITHUB_TOKEN not work here? GitHub has actually made the GITHUB_TOKEN more customizable, so we should switch to using that rather than this long-lived PAT (not required for this PR)

Reply:
GITHUB_TOKEN works here, good call!